wrims/bnd.bnd

```diff
@@ -0,0 +1,137 @@
+Bundle-SymbolicName: wrimsv2_plugin;singleton:=true
+Bundle-Name: Wrimsv2_plugin
```

Comment:
Is the 'W' in 'Wrimsv2_plugin' vs 'wrimsv2_plugin' intentional?

Reply:
This was intentional to explicitly match the contents of the original wrims-plugin jar, yes.